Dockerfile

```diff
@@ -125,11 +125,11 @@ RUN echo "build --spawn_strategy=standalone --genrule_strategy=standalone" \
     >>/etc/bazel.bazelrc
 
 # Put cuda libraries to where they are expected to be
-RUN mkdir /usr/local/cuda/lib &&  \
-    ln -s /usr/lib/x86_64-linux-gnu/libnccl.so.2 /usr/local/cuda/lib/libnccl.so.2 && \
-    ln -s /usr/include/nccl.h /usr/local/cuda/include/nccl.h && \
-    ln -s /usr/local/cuda/lib64/stubs/libcuda.so /usr/local/cuda/lib64/stubs/libcuda.so.1 && \
-    ln -s /usr/include/cudnn.h /usr/local/cuda/include/cudnn.h
+##RUN mkdir /usr/local/cuda/lib &&  \
+##    ln -s /usr/lib/x86_64-linux-gnu/libnccl.so.2 /usr/local/cuda/lib/libnccl.so.2 && \
+##    ln -s /usr/include/nccl.h /usr/local/cuda/include/nccl.h && \
+##    ln -s /usr/local/cuda/lib64/stubs/libcuda.so /usr/local/cuda/lib64/stubs/libcuda.so.1 && \
+##    ln -s /usr/include/cudnn.h /usr/local/cuda/include/cudnn.h
```

Comment:
Was leaving this commented out instead of removing it done on purpose?

Reply:
Yes, until we could confirm it was 100% okay